GANDLF/cli/config_generator.py

```diff
@@ -59,7 +63,7 @@ def generate_new_configs_from_key_and_value(
     return configs_to_return
 
 
-def remove_duplicates(configs_list):
+def remove_duplicates(configs_list: list) -> list:
     """
     Remove duplicate configs from a list of configs.
```

Comment:
Here as previously, I would opt for annotating list in a format List[object,type]

Reply:
Done.